pmp-reactjs-ui/src/auth/AppRoutes.js

```diff
@@ -20,8 +20,8 @@ function AppRoutes() {
           element: <GuardedRoute><MainLayout><Dashboard/></MainLayout></GuardedRoute>,
         },
         {
-          path: 'partnerCertificate',
-          element: <GuardedRoute><MainLayout><PartnerCertificate/></MainLayout></GuardedRoute>,
+          path: 'PartnerCertificatesList',
+          element: <GuardedRoute><MainLayout><PartnerCertificatesList/></MainLayout></GuardedRoute>,
```

Comment:
path should not change only component name

Reply:
Done.